src/main/java/at/hannibal2/skyhanni/utils/PetUtils.kt

```diff
@@ -176,9 +178,18 @@ object PetUtils {
     fun levelToXp(level: Int, petInternalName: NeuInternalName): Double? = runCatching {
         val rarityOffset = getRarityOffset(petInternalName) ?: return null
         if (level < 0 || level > getMaxLevel(petInternalName)) return null
-        return getFullLevelingTree(petInternalName)
-            .slice(0 + rarityOffset..<level + rarityOffset - 1)
-            .sumOf { it.toDouble() }
+        val levelTree = getFullLevelingTree(petInternalName)
+        if (rarityOffset + level > levelTree.size) {
+            if (lastRepoWarning.passedSince() > 5.minutes) {
+                ChatUtils.userError(
+                    "§cFailed to load pet levels from NEU repo. " +
+                        "§cYou can try to fix this by running `§e/${ItemUtils.resetCommand}`§c."
+                )
+                lastRepoWarning = SimpleTimeMark.now()
+            }
+            return null
+        }
+        return levelTree.subList(rarityOffset, rarityOffset + level).sumOf { it.toDouble() }
```

Comment:
```suggestion
        return levelTree.subList(rarityOffset, rarityOffset + level - 1).sumOf { it.toDouble() }
```

I think this is missing